DevIncubator-Autopark/MyQueue.cs

```diff
@@ -0,0 +1,109 @@
+﻿namespace DevIncubatorAutopark
+{
+    internal class MyQueue<T>
+    {
+        private const int _defaultSize = 10;
+        private T[] _queueArr;
+        private int _length;
+        private int _startIndex;
```

Comment:
Do you really need _startIndex? He's always `0`

Reply:
fixed